src/Conversations/EmailConversation.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace HelpScout\Api\Conversations;
+
+class EmailConversation extends Conversation
```

Comment:
With the introduction of typed `Conversation` objects like this, a further enhancement to this would be to [update fetching a list of conversations](https://github.com/helpscout/helpscout-api-php/blob/v2/examples/conversations.php#L37) to return these typed objects rather than generic `Conversation`.

This isn't something I think we should hold up this PR on though, too much good stuff here.